frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx

```diff
@@ -0,0 +1,69 @@
+import {
```

Comment:
this file is almost same as the index file of `Crews` components. Could we abstract a common component in order to decrease the  duplicate code?